src/main.ts

```diff
@@ -86,33 +85,31 @@ const clientParams = {
 
 async function run(): Promise<void> {
   try {
-    console.log(`Current branch: ${branchNameHead}.`)
-    console.log(commandToRun)
-    safeExec(commandToRun)
+    safeExec(`/usr/bin/git fetch`)
+    safeExec(`/usr/bin/git checkout ${branchNameBase}`)
```

Comment:
@tavisatjane I need this checkout before to base before doing `--changedSince` otherwise the jest diff command fails. The `fetch` alone isn't enough for some reason.